unix/x0vncserver/XDesktop.cxx

```diff
@@ -113,6 +114,10 @@ XDesktop::XDesktop(Display* dpy_, Geometry *geometry_)
       ledState |= 1u << i;
   }
 
+  // Initialize our list for newly added keysyms
+  added_keysyms.resize(256);
+  fill(added_keysyms.begin(), added_keysyms.end(), NoSymbol);
```

Comment:
Why not a `std::map` like the pressed keys? I assume this vector will be mostly unused anyway as most keycodes are already taken.